### PR TITLE
Include regex captured groups in docs

### PR DESCRIPTION
**Pull request by max-sixty**

### CODE REVIEW

File: docs/reference.md [hunk 1]
```diff
@@ -182,6 +182,8 @@ The following placeholders are supported:
 * `{{prefix}}` (only valid for `tag-name` / `tag-message`): The value prepended to the tag name.
 * `{{tag_name}}` (only valid for `tag-message`): The name of the git tag.
 
+Regex patterns such as `$1`, are also valid for referring to captured groups.
```
Comment (max-sixty) [inline docs/reference.md:185]:
```suggestion
Regex patterns, such as `$1`, are also valid for referring to captured groups.
```

File: docs/reference.md [hunk 2]
```diff
@@ -182,6 +182,8 @@ The following placeholders are supported:
 * `{{prefix}}` (only valid for `tag-name` / `tag-message`): The value prepended to the tag name.
 * `{{tag_name}}` (only valid for `tag-message`): The name of the git tag.
 
+Regex patterns, such as `$1`, are also valid for referring to captured groups.
```
Comment (epage) [inline docs/reference.md:185]: Doesn't this belong under the pre-release replacement section, under `replace`?

Comment (max-sixty) [reply docs/reference.md]: Yes, sorry, moving a bit too quickly there

**Comment by epage** [comment]
Thanks!

**Pull request merged**